nav2_dwb_controller/dwb_core/src/dwb_local_planner.cpp

```diff
@@ -86,6 +86,8 @@ void DWBLocalPlanner::configure(
     rclcpp::ParameterValue(std::string("dwb_plugins::SimpleGoalChecker")));
   declare_parameter_if_not_declared(node_, dwb_plugin_name_ + ".transform_tolerance",
     rclcpp::ParameterValue(0.1));
+  declare_parameter_if_not_declared(node_, dwb_plugin_name_ + ".short_circuit_trajectory_evaluation",
```

Comment:
Can you add this to the configuration files / documentation? (also what does this do?)

Reply:
Funny though. Its been used in a if statement which is empty. I think we should remove it for now
```c++
score.total += critic_score * cs.scale;
    if (short_circuit_trajectory_evaluation_ && best_score > 0 && score.total > best_score) {
      // since we keep adding positives, once we are worse than the best, we will stay worse
      break;
    }
I don't seen any use. (may be in future updates)
```

Reply:
Its not empty -- it breaks out of the for loop, that has value.

Reply:
Oh! my bad. But I couldn't pin point what it exactly does. But breaks the loop when total scores goes above the best score.

Reply:
One more question, which documentation I should update? I know the params file but don't know about the documentation file.

Reply:
https://github.com/ros-planning/navigation2/tree/master/sphinx_doc/configuration